src/clients/Elsa.Api.Client/Resources/WorkflowInstances/Contracts/IWorkflowInstancesApi.cs

```diff
@@ -62,6 +62,22 @@ public interface IWorkflowInstancesApi
     [Post("/bulk-actions/delete/workflow-instances/by-id")]
     Task BulkDeleteAsync(BulkDeleteWorkflowInstancesRequest request, CancellationToken cancellationToken);
 
+    /// <summary>
+    /// Cancels a workflow instance.
+    /// </summary>
+    /// <param name="id">The ID of the workflow instance to cancel.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    [Post("/cancel/workflow-instances/{id}")]
+    Task CancelAsync(string id, CancellationToken cancellationToken = default);
+    
+    /// <summary>
+    /// Cancels a list of workflow instances.
+    /// </summary>
+    /// <param name="request">The request containing the selection of workflow instances to cancel.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    [Post("/bulk-actions/cancel/workflow-instances")]
```

Comment:
I thinks that is not important to specefie that is a bulk action,
[Post("/cancel/workflow-instances")] is enough.

Reply:
Agreed, I'll remove the bulk-actions part from the url (and other existing urls)